File: Algorithms/728.Self-Dividing-Numbers/self_dividing_numbers.py
class Solution:
    def selfDividingNumbers(self, left: int, right: int) -> List[int]:
        res = []
        for ele in range(left, right + 1):
            temp = ele
            flag = True
            while temp:
                remain = temp % 10
                temp = temp // 10
                if remain == 0 or ele % remain != 0:
                    flag = False
                    break
            if flag:
                res.append(ele)
        return res

File: Algorithms/728.Self-Dividing-Numbers/self_dividing_numbers.go
func selfDividingNumbers(left int, right int) []int {
    res := []int{}
    for i := left; i <= right; i++ {
        temp := i
        flag := true
        for temp > 0 {
            remain := temp % 10
            temp = temp / 10
            if remain == 0 || i % remain > 0 {
                flag = false
                break
            }   
        }
        if flag {
            res = append(res, i)
        }
    }
    return res
}